contrib/database/sql/sql.go

```diff
@@ -12,49 +12,169 @@
 package sql
 
 import (
+	"context"
 	"database/sql"
 	"database/sql/driver"
 	"errors"
+	"math"
+	"reflect"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql/internal"
 )
 
+// registeredDrivers holds a registry of all drivers registered via the sqltrace package.
+var registeredDrivers = &driverRegistry{
```

Comment:
I wonder if this would look better throughout as being called simply `drivers` instead of `registeredDrivers`. Maybe the prefix is too much?

Reply:
I'm mixed on this. There's already lots of references to `driver.Driver`, so it might be confusing for this specific case. `registry` could work, but sounds like it'd be a package instead of a variable.

Reply:
Sure, as you wish.